router/server.go

```diff
@@ -191,6 +201,13 @@ func (s *Server) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 		http.Error(w, "", http.StatusTeapot)
 		return
 	}
+
+	// This redirects all HTTP request to use HTTPS for all connections which are not local to the server.
+	if (s.configHttp.RedirectHTTP && !isSecure && req.Proto == "HTTP/1.1" && req.Host != "127.0.0.1" && req.Host != "localhost") {
+		http.Redirect(w, req, "https://" + req.Host + req.URL.String(), http.StatusMovedPermanently)
```

Comment:
I recall that as well. Will change it, thanks
The information contained in this communication from the sender is confidential. It is intended solely for use by the recipient and others authorized to receive it. If you are not the recipient, you are hereby notified that any disclosure, copying, distribution or taking action in relation of the contents of this information is strictly prohibited and may be unlawful. Please notify the sender by replying to this message and then delete it from your system. Please report email abuse / misuse to IMQS Software<www.imqs.co.za>.